endpointmanager/cmd/CHPLreport/main.go

```diff
@@ -23,34 +23,42 @@ func main() {
 	// get report data and export to tmp/export.csv
 	getCHPLEndpointsStatement := `
 	COPY (
-		SELECT
+        SELECT DISTINCT
    			f.list_source AS service_base_url,
    			f.url as endpoint,
+			vendors.name as developer,
    			CASE WHEN COUNT(metadata.http_response)=0
         		THEN NULL
         		ELSE ROUND(100 - COUNT(NULLIF(metadata.http_response, 200))::numeric/COUNT(metadata.http_response) * 100,2)
-   			END AS server_status_days,
-   			string_agg(DISTINCT NULLIF(metadata.http_response,200)::varchar, ',') AS server_status,
+   			END AS server_status_percent,
```

Comment:
I think both of the percentages are backwards. It looks like in the reference CSV, it should be the percentage of time that it is found to have the issue, instead of the percent of time it's been passing?